pokemongo_bot/cell_workers/move_to_map_pokemon.py

```diff
@@ -168,7 +168,7 @@ def get_pokemon_from_social(self):
 
     def get_pokemon_from_url(self):
         try:
-            request = requests.get(self.config['address'])
+            request = requests.get('{}/{}?gyms=false&scanned=false'.format(self.config['address'], self.map_path))
```

Comment:
That's not a bug. 'map_path' was deprecated by me, if that's what youre referring.